tensorflow/lite/g3doc/guide/build_cmake.md

```diff
@@ -290,6 +290,8 @@ cmake --build . -j
 
 This command generates the following shared library in the current directory.
 
+**Note:** On Windows system, you can find the `tensorflowlite_c.dll` under `debug` directory.
```

Comment:
what about release build?

Reply:
I am the user that opened the underlying issue.
There is no binary being build. Running a search on Windows for any `.dll` in the build directory does not lead to any files.

Reply:
You mean release build doesn't generate any binary?

Reply:
Yes, I tried it on my machine and also on a friend's. On Mac and Linux however it works.